internal/controller/suite/resources/suite_test.go

```diff
@@ -87,7 +87,7 @@ var _ = BeforeSuite(func() {
 	defer func(zapLog *uberzap.Logger) {
 		_ = zapLog.Sync()
 	}(zapLog)
-	log = zapr.NewLogger(zapLog)
+	log = zapr.NewLogger(zapLog).WithSink(GinkgoLogr.GetSink())
```

Comment:
I added this as I found something similar recommended in the Kubebuilder Book a while back. The idea is that Ginkgo will swallow log output unless there is a spec failure. Of course, this is kind of moot given the make target passes along a `-vv` flag. I'm fine removing this if it is undesirable in your view.

Reply:
https://book.kubebuilder.io/reference/envtest